test/libsolidity/SolidityScanner.cpp

```diff
@@ -710,6 +710,34 @@ BOOST_AUTO_TEST_CASE(irregular_line_breaks_in_strings)
 	}
 }
 
+BOOST_AUTO_TEST_CASE(solidity_keywords)
+{
+	// These are tokens which have a different meaning in Yul.
+	Scanner scanner(CharStream("return byte bool address var in true false leave", ""));
```

Comment:
Maybe keep `"return byte bool address var in true false leave"` in a separate variable so a typo won't mess up the tests?

Reply:
@axic still this

Reply:
I also agree with this - not nice to have the string duplicated like this.

Reply:
Changed, thought a lot of other test cases replicate this style.